### DISCUSSION TITLE
Update: ShipFromSupplySourceId attribute adeded to Order feeds attributes

### DISCUSSION THREAD

**Discussion by andiec1** [amzn/selling-partner-api-models#3884] [Announcements]
The `ShipFromSupplySourceId` attribute has been added to the following Order feeds: 

- [`POST_ORDER_FULFILLMENT_DATA`](https://developer-docs.amazon.com/sp-api/docs/order-feeds-attributes#post_order_fulfillment_data)
- [`POST_FLAT_FILE_FULFILLMENT_DATA`](https://developer-docs.amazon.com/sp-api/docs/order-feeds-attributes#post_flat_file_fulfillment_data)

**Comment by bullfrogsoftware** [comment]
Where is documentation instructing us on how to gather the correct ID to enter in the 'ShipFromSupplySourceId' attribute?

**Reply by tang226** [reply]
The shipFromSupplySourceId in the payload is the S2 ID copied from Settings -> Shipping Setting -> Locations, choose one location, click.try S2 ID or S2 Code~
![351421721184247_ pic](https://github.com/user-attachments/assets/aa611322-b833-433b-aaa7-3f4d6fe081d3)
